Readability.js

```diff
@@ -1722,6 +1722,22 @@ Readability.prototype = {
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
 
+    // Get <div> nodes which have <br> node(s) and append them into the `nodes` variable.
+    // Some articles' DOM structures might look like
+    // <div>
+    //   Sentences<br>
+    //   <br>
+    //   Sentences<br>
+    // </div>
+    var brNodes = this._getAllNodesWithTag(this._doc, ["div > br"]);
+    if (brNodes.length) {
+      var set = new Set();
+      [].forEach.call(brNodes, function(node) {
+        set.add(node.parentNode);
+      });
+      nodes = [].concat.apply(Array.from(set), nodes);
```

Comment:
Looks like the new idea might not be faster. I would say they are almost same.

I run 10 times performance test for each one on Firefox. The result is `5.952 ms` vs `6.12 `(origin vs new idea).

The below is the full test report.

|     | Origin | New Idea |
|-----|----------|----------|
| 1.  | 5.46 ms  | 6.09 ms  |
| 2.  | 6.00 ms  | 6.23 ms  |
| 3.  | 6.01 ms  | 6.13 ms  |
| 4.  | 6.01 ms  | 5.91 ms  |
| 5.  | 5.76 ms  | 6.53 ms  |
| 6.  | 5.63 ms  | 6.04 ms  |
| 7.  | 5.75 ms  | 6.15 ms  |
| 8.  | 6.22 ms  | 5.51 ms  |
| 9.  | 6.55 ms  | 6.41 ms  |
| 10. | 6.13 ms  | 6.20 ms  |
| AVG | 5.952 ms | 6.12 ms  |